lib/config/executor/container_vm.go

```diff
@@ -44,6 +44,21 @@ type Common struct {
 	Notes string `vic:"0.1" scope:"hidden" key:"notes"`
 }
 
+// Common data (specifically for a containerVM) between managed entities, across execution environments.
+type Common2 struct {
```

Comment:
What is the thought process behind naming this Common2, I just want to be sure that there isn't a better or more appropriate name for this.